serde/de.py

```diff
@@ -878,6 +886,9 @@ def literal(self, arg: DeField) -> str:
 
     def default(self, arg: DeField, code: str) -> str:
         exists = f'"{arg.conv_name()}" in {arg.datavar}'
+        if arg.alias:
+            aliases = map(lambda s: f'"{s}"', [arg.name, *arg.alias])
+            exists += f' or _exists_by_aliases({arg.datavar}, [{",".join(aliases)}])'
```

Comment:
I saw the generated code for the following `Foo` class. 

```python
    @serde.serde
    class Foo:
        a: int = serde.field(default=2, alias=["b", "c", "d"])
```

Here is the code excerpt from `from_dict`.
```python
    __a = (
        (_get_by_aliases(data, ["a", "b", "c", "d"]))
        if "a" in data or _exists_by_aliases(data, ["a", "b", "c", "d"])
        else serde_scope.defaults["a"]
    )
```

I think "a" is checked twice? Maybe we can further optimize to generate the code like this? 🤔  
```python
    __a = (
        (_get_by_aliases(data, ["a", "b", "c", "d"]))
        _exists_by_aliases(data, ["a", "b", "c", "d"])
        else serde_scope.defaults["a"]
    )
```

Reply:
I tested this works 👍 
```python
    def default(self, arg: DeField, code: str) -> str:
        if arg.alias:
            aliases = map(lambda s: f'"{s}"', [arg.name, *arg.alias])
            exists = f'_exists_by_aliases({arg.datavar}, [{",".join(aliases)}])'
        else:
            exists = f'"{arg.conv_name()}" in {arg.datavar}'

```

Reply:
Thanks!  I pushed your version.